seqr/utils/communication_utils.py

```diff
@@ -29,6 +34,49 @@ def _post_to_slack(channel, message):
     )
     return response.raw
 
+def safe_upload_gcs_file_to_slack(channel_id, gcs_file_path, uploaded_filename, message):
+    try:
+        _upload_gcs_file_to_slack(channel_id, gcs_file_path, uploaded_filename, message)
+    except Exception as e:
+        logger.error('Slack error: {}: Original message in channel ({}) - {}'.format(e, channel_id, message))
+
+
+def _upload_gcs_file_to_slack(channel_id, gcs_file_path, uploaded_filename, message):
+    file_content = download_gcs_file_as_bytes(gcs_file_path)
+
+    # https://api.slack-gov.com/messaging/files#uploading_files
+    # Note: slackbot needs files:read and files:write scopes
```

Comment:
I made a new bot for testing, but we can reuse the existing bot as long as it has these scopes. 

The message _will_ come from the 'Slack API Tester' (the name of the bot app) so maybe we rename it when the scopes are added (which I believe @hanars has to do because only the slack app creator can change these values, or I can be added as an authorized app member and possibly do that).